gyrinx/core/templates/core/includes/list.html

```
@@ -13,237 +13,231 @@
+                            <span data-bs-toggle="tooltip"
+                                  data-bs-title="This list is visible to all users"
+                                  class="tooltipped">Public</span>
+                        {% endif %}
+                        {% if list.owner_cached == user and not list.public %}
+                            <i class="bi-eye-slash"></i>
+                            <span data-bs-toggle="tooltip"
+                                  data-bs-title="This list can only be accessed by users with the direct link"
+                                  class="tooltipped">Unlisted</span>
+                        {% endif %}
+                    </div>
+                    {% if list.owner_cached == user and list.archived_fighters_cached.count > 0 %}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)